modules/vslam/sba.cpp

```diff
@@ -0,0 +1,579 @@
+#include <opencv2/opencv.hpp>
```

Comment:
all-in-one `opencv2/opencv.hpp` header is deprecated and should not be used in OpenCV sources/samples